### PR TITLE
DWR-1215 Migrate district/school groups

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml [hunk 1]
```diff
@@ -442,13 +466,129 @@ sql:
...
+             warehouseRead: >-
+              SELECT
+                id,
+                name,
+                natural_id,
+                external_id,
+                ? as migrate_id
+              FROM district_group wdg
+               WHERE EXISTS( SELECT id FROM school  WHERE created > ? AND created <= ? AND deleted = 0 AND district_group_id = wdg.id)
+              INTO OUTFILE S3 '${archive.root}/${migrate.aws.location}/dstrgroup'
```
Comment (agorina) [inline migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: Using `district_group` as a file name causes a conflict with the '`district`. I am not sure why but suspect that it goes by the first X chars?

Comment (malaffoon) [reply migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: That sucks. But i'd vote for a different contraction, per just `dgroup` and `sgroup`?

Comment (chdietz) [reply migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: Is the conflict on the write or the load?
Wonder if something is matching only up to the underscore?

Comment (agorina) [reply migrate-olap/src/main/resources/application.warehouse.to.staging.sql.yml]: The conflict is on the write. And I have tried without the underscore, no luck.

**Pull request merged**